Description: Disable the deprecated hyper-requester feature
 and avoid some big migrations
Bug: https://gitlab.com/schmiddi-on-mobile/railway-backend/-/issues/15
Forwarded: not-needed
Author: Blair Noctis <ncts@debian.org>
Last-Update: 2025-04-01
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -49,3 +49,3 @@
 
-[dependencies.hyper]
+[disabled.dependencies.hyper]
 version = "0.14"
@@ -53,3 +53,3 @@
 
-[dependencies.hyper-rustls]
+[disabled.dependencies.hyper-rustls]
 version = "0.24"
@@ -65,3 +65,3 @@
 
-[dependencies.rustls]
+[disabled.dependencies.rustls]
 version = "0.21"
@@ -69,3 +69,3 @@
 
-[dependencies.rustls-pemfile]
+[disabled.dependencies.rustls-pemfile]
 version = "2.1"
@@ -82,8 +82,8 @@
 [features]
-hyper-requester = [
-    "hyper-rustls",
-    "hyper",
-    "rustls",
-    "rustls-pemfile",
-]
+#hyper-requester = [
+#    "hyper-rustls",
+#    "hyper",
+#    "rustls",
+#    "rustls-pemfile",
+#]
 polylines = ["geojson"]
